test/regress/regress0/datatypes/issue2838.cvc

```diff
@@ -0,0 +1,14 @@
+% EXPECT: sat
```

Comment:
Can we maybe get the value for `x` and check whether it matches what we expect it to be?

Reply:
I've discussed this offline with @ajreynol and he pointed out that current master fails the regression because `--check-models` detects a non-constant value. Thus, getting the value does not add anything here.